mcodingbot/config.py

```diff
@@ -24,6 +24,10 @@ class Config:
     patron_role: int = -1
     donor_role: int = -1
 
+    # capacity, cooldown
+    message_sent_cooldown: tuple[float, float] = (1, 60)
```

Comment:
I think it is confusing what the `message_sent_cooldown` is for
```suggestion
    highlight_message_sent_cooldown: tuple[float, float] = (1, 60)
```